src/main/java/org/assertj/core/internal/Longs.java

```diff
@@ -62,4 +66,20 @@ public void assertIsCloseTo(AssertionInfo info, Long actual, Long expected, Offs
     if (absDiff > offset.value) throw failures.failure(info, shouldBeEqual(actual, expected, offset, absDiff));
   }
 
+    @Override
+    public void assertIsCloseToPercentage(AssertionInfo info, Long actual, Long other,
+                                                    Percentage<Long> percentage) {
+        assertNotNull(info, actual);
+        checkPercentageIsNotNull(percentage);
+        checkNumberIsNotNull(other);
+
+        if (org.assertj.core.util.Objects.areEqual(actual, other)) return;
+
+        Offset<Double> calculatedOffset = offset(percentage.value * other / 100d);
+
+        Long absDiff = abs(other - actual);
+        if (absDiff > calculatedOffset.value) throw failures.failure(info, ShouldBeEqualWithinPercentage
+            .shouldBeEqualWithinPercentage(actual, other, percentage, absDiff));
```

Comment:
static import `shouldBeEqualWithinPercentage`, it makes the code more readable
applies to the other classes